.gitignore

```diff
@@ -1,3 +1,5 @@
 .DS_Store
 node_modules
 version.json
+.idea/
+umd/
```

Comment:
Why do you want to exclude built files?

Reply:
That's a mistake, please exclude that line